src/model/user.rs

```diff
@@ -407,7 +411,7 @@ impl User {
     ///
     /// client.on_message(|_, msg| {
     ///     if msg.content == "~help" {
-    ///         let url = CACHE.read().unwrap().user.invite_url(Permissions::empty());
+    ///         let url = CACHE.read().unwrap().user.invite_url(Permissions::empty()).unwrap();
```

Comment:
I'd like to avoid non-lock unwraps in future documentation (and slowly fixing them in old documentation). Maybe we could match this.. or push the invite text in an if-let only if this is Ok? It's unfortunate event handlers don't return a Result right now to make use of `?`.